src/libmongoc/doc/mongoc_uri_t.rst

```diff
@@ -283,6 +283,7 @@ MONGOC_URI_SAFE                            safe                              {tr
 
     mongoc_uri_copy
     mongoc_uri_destroy
+    mongoc_uri_finalize_options
```

Comment:
I think this `mongoc_uri_finalize_options.rst` is missing from the PR.

Reply:
Just added. Thanks for catching that.